### PR TITLE
Feat/translate texts into japanese by using deepl

### PR DESCRIPTION
**Pull request by yuki-o1o5**
## Main Proposed Changes
Update main/index.jsx, main/[id].jsx, all/index.jsx, and all/[id].jsx in order to translate texts into Japanese

### Code changes and Reasons
- Install deepl-node in order to use deepl api
- Create translateToJapanese functions in order to translate data which is fetched by hacker news api into Japanese.
- Change style in order to be easy-read (Add)
- Create helpers in order to be readable code (Add)

### Related Issues
- #8 

## Screen Cast
https://user-images.githubusercontent.com/108368307/210117478-43637c7c-290e-4e63-b923-4edcd84646e6.mov

## Screen Cast (Add)
https://user-images.githubusercontent.com/108368307/210117839-6ec337c2-6461-4fb7-8b47-8faf6be72ece.mov

### CODE REVIEW

File: pages/all/[id].jsx [hunk 1]
```diff
@@ -1,101 +1,127 @@
...
-            detailArticleCategoryTitle={"Article Summary"}
+            detailArticleCategoryTitle={"Story URL"}
           />
           <div className="main_text-container">
-            <DetailArticleText
-              detailArticleText={
-                "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Tempor nec feugiat nisl pretium. Purus non enim praesent elementum facilisis. Ultrices sagittis orci a scelerisque purus semper eget duis. At lectus urna duis convallis convallis tellus. Gravida in fermentum et sollicitudin ac orci phasellus egestas. Ut sem nulla pharetra diam sit. Id donec ultrices tincidunt."
-              }
-            />
+            <DetailArticleUrl DetailArticleUrl={japaneseStoryDetail.url} />
```
Comment (isamisushi) [inline pages/all/[id].jsx]: minor: the name of the prop should be camelcase.

File: helpers/deepl/common.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const JA = "ja";
```
Comment (isamisushi) [inline helpers/deepl/common.js]: minor: You can place this variable in `ROOT/constants/deepl.js`.

**Review by isamisushi — APPROVED** [review]
LGTM! You can merge it after fixing minor things

File: pages/all/[id].jsx (hunk 1 above)
Comment (yuki-o1o5) [reply pages/all/[id].jsx]: I understand. 
Changed the name -->[link](https://github.com/yuki-o1o5/hacker-news-in-japanese/commit/4b98b3a296d89d7d1121ac4d40a99c3ed1adf814)

File: helpers/deepl/common.js (hunk 2 above)
Comment (yuki-o1o5) [reply helpers/deepl/common.js]: I understand.
Changed the place of the variable. -->[link](https://github.com/yuki-o1o5/hacker-news-in-japanese/commit/a75e264c6b0b1ea84b1ac44fcad2b28a8ef04502)

**Pull request merged**